src/compiler/compiler.js

```diff
@@ -52,29 +51,29 @@ compiler.compileEncoding = function (encoding, stats) {
       padding: 'auto',
       data: compiler.data(encoding),
       // global scales contains only time unit scales
-      scales: compiler.time.scales(encoding)
+      scales: compiler.time.scales(encoding),
+      marks: [{
+        _name: 'cell',
```

Comment:
Thanks, great that you comment.  I forgot to jot this down in the issue tracker!  
Now fixed in #596